docs/featuregate.yaml

```diff
@@ -5,5 +5,7 @@ apiVersion: config.openshift.io/v1
 kind: FeatureGate
 metadata:
   name: open-cluster-management-discovery
+  labels:
+    open-cluster-management: ''
```

Comment:
I think the label would be better as something like:

`console.open-cluster-management.io`

Reply:
Should the label be updated here as well? - https://github.com/open-cluster-management/console/blob/main/backend/src/routes/watch.ts#L80

Reply:
Yeah, but does the FG created currently for discovery add the label?  I know you guys are trying to GA discovery, so the FG won't be needed anyways

Reply:
Well the user would create it manually when enabling discovery as of now. Since the label in the `watch` is a new addition, we would like to get our documented feature gate up to speed first.

But you are right in the long term we do plan to get rid of the feature gate. This PR will also help ensure our examples are up to date to help with doc/QE validation in the meantime.

Reply:
@zkayyali812 sounds good, let's make the update then.  Thanks for contributing these updates

Reply:
@showeimer Np! I just pushed an update to contain this suggestion